pkg/backend/filestate/backend.go

```diff
@@ -253,8 +265,28 @@ func (b *localBackend) SupportsOrganizations() bool {
 	return false
 }
 
-func (b *localBackend) ParseStackReference(stackRefName string) (backend.StackReference, error) {
-	return localBackendReference{name: tokens.Name(stackRefName)}, nil
+func (b *localBackend) ParseStackReference(stackRef string) (backend.StackReference, error) {
+	var name, project string
+	split := strings.Split(stackRef, "/")
+	switch len(split) {
+	case 1:
+		name = split[0]
+	case 2:
+		project = split[0]
+		name = split[1]
+	default:
+		return nil, fmt.Errorf("could not parse stack reference '%s'", stackRef)
+	}
+
+	if project != "" && !tokens.IsName(project) {
+		return nil, errors.New("project names may only contain alphanumerics, hyphens, underscores, and periods")
```

Comment:
Include here in the error message the given `project` name and/or `stackRef` it was parsed from.